charts/folder-config-connector/templates/folder.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.name (not .Values.folders) }}
+---
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Folder
+metadata:
+  labels:
+  {{- include "folder-config-connector.labels" . | nindent 4 }}
+    {{- if .Values.labels }}
+    {{- toYaml .Values.labels | nindent 4 }}
+  {{- end  }}
+  {{- with .Values.annotations  }}
+  annotations:
+    {{- toYaml . | nindent 4 }}  
+  {{- end  }}
+  name: {{ .Values.name | quote }}
+spec:
+  {{- if .Values.displayName }}
+  displayName: {{ .Values.displayName | quote }}
```

Comment:
this is requeired field, if not set validation should throw an error. if makes no sense for me

Reply:
moreover there is a lot of requirements that displayName must meet:

```The display name must start and end with a letter or digit, may contain letters, digits, spaces, hyphens and underscores and can be no longer than 30 characters.```

but it can be omitted for me.

Reply:
changed to required field